cluster-sync/rook-ceph/pool.yaml

```diff
@@ -1,15 +1,22 @@
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: replicapool
+  namespace: rook-ceph
+spec:
+  failureDomain: host
+  replicated:
+    size: 1
```

Comment:
We want the size to be the number of nodes in the cluster, so this is the reason we had the CephBlockPool in the .sh file where we could substitute the value based on the KUBEVIRT_NUM_NODES environment variable.

Reply:
True, its not that important.